docker/llm/README.md

```diff
@@ -1,29 +1,43 @@
-## Getting started with BigDL LLM on Windows
+## Getting started with BigDL LLM on cpu
```

Comment:
The title is confusing; should be something like "Docker user guide for BigDL-LLM on CPU"